bao_supplier_certificate/data/mail_template.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="email_template_supplier_certificate" model="mail.template">
+        <field name="name">Supplier Certificate Email</field>
+        <field
+            name="model_id"
+            ref="bao_supplier_certificate.model_bao_supplier_certificate"
```

Comment:
I'm not sure the prefix `bao_supplier_certificate` is needed here.

Reply:
I'm not sure either. But it cannot hurt to let it there.